internal/graph/dotgraph_test.go

```diff
@@ -326,6 +326,46 @@ func TestTagCollapse(t *testing.T) {
 	}
 }
 
+func TestEscapeDoubleQuotes(t *testing.T) {
+	for _, tc := range []struct {
+		desc  string
+		input string
+		want  string
+	}{
+		{
+			desc:  "with multiple doubles quotes",
+			input: `label: "foo" and "bar"`,
+			want:  `label: \"foo\" and \"bar\"`,
+		},
+		{
+			desc:  "with already escaped doubles quotes",
+			input: `label: \"foo\" and \"bar\"`,
+			want:  `label: \"foo\" and \"bar\"`,
+		},
+		{
+			desc:  "with escaped character but no doubles quotes",
+			input: `label: \$foo and bar`,
+			want:  `label: \$foo and bar`,
+		},
+		{
+			desc:  "with escaped backslash but not escaped doubles quotes",
+			input: `label: \\"foo" and bar`,
```

Comment:
To clarify this example:

I expect that whatever is specified in the input label string (that is, in the profile comment field) will get visualized verbatim in the final rendered output.

From this point of view I think I'd expect `\\` to become `\\\\` so that in the final rendered version it becomes two backslashes.

Am I missing something?

Reply:
That does make sense, just that the regex that I had was based on the idea of keeping the previous behavior intact. Previously if you had a \\\\ in a comment, it would show up as \\ in the graph.